l10n_th_address/__manifest__.py

```diff
@@ -32,6 +32,5 @@
     ],
     'installable': True,
     'auto_install': False,
+    'post_init_hook': '_update_data_noupdate',
```

Comment:
The beta error is emitted:
 - `l10n_th_address/__manifest__.py:4: [C8101(manifest-required-author), ] Missing author required "Odoo Community Association (OCA)" in manifest file`:
   - https://travis-ci.org/OCA/l10n-thailand/jobs/228809869#L514

But we have these checks as beta yet.
I'll remove it from beta to affect the build result.